apps/exploits/api.py

```diff
@@ -16,17 +16,15 @@
 from rest_framework.response import Response
 from rest_framework.views import APIView
 
-from osidb.models import Affect, Flaw, Tracker
```

Comment:
This is being added and removed in the same PR. Please use squash or fixup to keep the history clean so we can easily review what has really changed. I am pausing my review here until it is done as I see that I already spent time commenting stuff in the previous commit which is no more existing in this one.